### DISCUSSION TITLE
Quick question regarding github.xiaofsu.com

### DISCUSSION THREAD

**Discussion by cybrox** [community/community#23481] [New to GitHub]
<p>Over the last few days, when I was googling for repos, I was sometimes seemingly logged in or logged out of GitHub.  I didn’t really think much of it until I just tried to log in to star a project and the login failed.</p>
<p>Turns out on multiple occasions, Google redirected me to <a href="https://github.xiaofsu.com/" rel="noopener nofollow ugc">https://github.xiaofsu.com/</a> instead of <a href="https://github.com" rel="noopener nofollow ugc">https://github.com</a>, which just seems to mirror GitHub 1:1.</p>
<p>Is this an elaborate phishing attempt, an inofficial mirror not blocked in China or something else entirely? I’m surprised to find zero information on that site anywhere, not even a mention in the community sections.</p>
<p>(I’ve changed my password and had 2FA active, so no big deal for me)</p>

**Comment by queenofcorgis** [comment, marked as answer]
<p>Hi there to report phishing to Google via <a href="https://safebrowsing.google.com/safebrowsing/report_phish/">report a phishing page</a>.</p>
